src/o365/spo/commands/list/list-contenttype-remove.ts

```diff
@@ -0,0 +1,209 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandTypes,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  contentTypeId: string;
+  confirm?: boolean;
+}
+
+class SpoListContentTypeRemoveCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_CONTENTTYPE_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes content type from list';
+  }
+
+  public types(): CommandTypes | undefined {
+    return {
+      string: ['contentTypeId', 'c']
+    };
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const resource: string = Auth.getResourceFromUrl(args.options.webUrl);
+    let siteAccessToken: string = '';
+
+    const removeContentTypeFromList: () => void = (): void => {
+      if (this.debug) {
+        cmd.log(`Retrieving access token for ${resource}...`);
+      }
+
+      auth
+        .getAccessToken(resource, auth.service.refreshToken as string, cmd, this.debug)
+        .then((accessToken: string): request.RequestPromise => {
+          siteAccessToken = accessToken;
+
+          if (this.debug) {
+            cmd.log(`Retrieved access token ${accessToken}.`);
+          }
+
+          if (this.verbose) {
+            const list: string = args.options.listId ? encodeURIComponent(args.options.listId as string) : encodeURIComponent(args.options.listTitle as string);
+            cmd.log(`Removing content type ${args.options.contentTypeId} from list ${list} in site at ${args.options.webUrl}...`);
+          }
+
+          let requestUrl: string = '';
+
+          if (args.options.listId) {
+            requestUrl = `${args.options.webUrl}/_api/web/lists(guid'${encodeURIComponent(args.options.listId)}')/ContentTypes('${encodeURIComponent(args.options.contentTypeId)}')`;
+          }
+          else {
+            requestUrl = `${args.options.webUrl}/_api/web/lists/GetByTitle('${encodeURIComponent(args.options.listTitle as string)}')/ContentTypes('${encodeURIComponent(args.options.contentTypeId)}')`;
+          }
+
+          const requestOptions: any = {
+            url: requestUrl,
+            method: 'POST',
```

Comment:
No need to specify `POST` here, since we're using `request.post` later on already.